com.odnoyko.valot.json

```diff
@@ -0,0 +1,73 @@
+{
+    "id" : "com.odnoyko.valot",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "49",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.node20",
+        "org.freedesktop.Sdk.Extension.typescript"
+    ],
+    "command" : "com.odnoyko.valot",
+    "finish-args": [
+	"--socket=wayland",
+	"--socket=fallback-x11",
+	"--device=dri",
+	"--filesystem=xdg-documents:create",
+	"--socket=cups"
+	],
```

Comment:
indentation is off here

<img width="499" height="429" alt="image" src="https://github.com/user-attachments/assets/3fcdf5b7-18c5-42a6-b0fd-5ed24bc883fe" />

Reply:
Indentation is wrong still.